src/migrations/1674664190143-sync.ts

```diff
@@ -1,40 +0,0 @@
-import { MigrationInterface, QueryRunner } from "typeorm";
-
-export class sync1674664190143 implements MigrationInterface {
```

Comment:
![2023-03-01 at 16 07](https://user-images.githubusercontent.com/38915833/222080695-b627ff59-74f6-40e0-9d4e-eb4c88366cf6.png)
we should have these migration files? why they are deleted.

Reply:
Done